### ISSUE TITLE
when adding a stutus and adding back ground and check-in the map doesn't show ?

### ISSUE THREAD

**Issue by spirogg** [PHPfox-Official/phpfox-v4-issues#2761]
----------------------------------------

## What's happened?
...  Hi when I try to add a status with Check-in and adding a background the Goggle Maps don;t show? anyway to add the Map also ?

## Steps to reproduce:
1. go to home page, click on check-in to add your location then add a status and add a background image. now you click share and the background and your status you wrote only shows not the google map ?
2. 
3. 

## What's expected?
... somehow you should be able to see the google map also would be nice

## Browsers and Devices tested
(Example: Chrome on iPhone X, Safari on Macbook, Miscrosoft Edge on Windown 10, Firefox on Ubuntu 16.04, ...)

...

## Server information
(Example: CentOS 7, php 7.1 apache)

...

## phpFox version
(Example: phpFox 4.6.0)

... 4.7.7

## Screenshots
...
see photos below
- 
![check-in doesnt show map when adding background](https://user-images.githubusercontent.com/17736688/63991471-a6512700-caad-11e9-8ef2-c8dd79884d53.PNG)
- 
![check-in doesnt show map when adding background2](https://user-images.githubusercontent.com/17736688/63991472-a6512700-caad-11e9-93c5-48e883cc1eac.PNG)

**Comment by spirogg** [comment]
can you fix it by adding the map and a link to **view map**, **hide map** like you have in Events app?

see photo below ?
- no map showing just the link for users to **Show Map**
![checkin with background](https://user-images.githubusercontent.com/17736688/63992509-d0a4e380-cab1-11e9-978a-b6e7a775b117.PNG)
- then the map shows and a link for users to **Hide Map** 
![checkin with background2](https://user-images.githubusercontent.com/17736688/63992510-d0a4e380-cab1-11e9-9d6a-497efde69ef9.PNG)

**Comment by harrison05** [comment]
> ... Hi when I try to add a status with Check-in and adding a background the Goggle Maps don;t show? anyway to add the Map also ?

Yes, this is our business rule, we hide maps to show status background. If show both of them, the layout on feed item is too long. 
About "Show map/Hide map" button on feed, our team will review and consider this.

Regards.
